lib/SPIRV/SPIRVReader.h

```diff
@@ -185,7 +185,8 @@ class SPIRVToLLVM {
   // A SPIRV value may be translated to a load instruction of a placeholder
   // global variable. This map records load instruction of these placeholders
   // which are supposed to be replaced by the real values later.
-  typedef std::map<SPIRVValue *, LoadInst *> SPIRVToLLVMPlaceholderMap;
+  typedef std::unordered_map<SPIRVValue *, LoadInst *>
```

Comment:
IMO, it may be worthwhile to first measure if this really improves the performance.

As far as I understand, this depends on how many load instructions of a global variable we are likely to have during an "average" translation - the hash map may actually be slower at low elements numbers, linear complexity outweighing only at higher elements number. @AlexeySachkov, what are your thoughts on this?

Reply:
Firstly, I accidentally changed definition for another type. I wanted to change SPIRVToLLVMLoopMetadataMap definition.
Secondly, I thought and agreed with you, better to leave it as is until performance improvement is required.